app/package.json

```diff
@@ -19,6 +19,7 @@
     "ts:check": "tsc --noEmit"
   },
   "dependencies": {
+    "@trussworks/react-uswds": "^4.0.0",
     "@uswds/uswds": "^3.1.0",
     "i18next": "^22.0.0",
     "next": "^13.0.1",
```

Comment:
We'll wanna bump this up to `^13.0.6` so that `next/link` supports the `className` prop

**In 13.0.1**

<img width="135" alt="CleanShot 2022-12-09 at 10 08 24@2x" src="https://user-images.githubusercontent.com/371943/206765611-231f7824-177b-4b10-a51d-aaa2b51a9a05.png">

**In 13.0.6**

<img width="122" alt="CleanShot 2022-12-09 at 11 27 18@2x" src="https://user-images.githubusercontent.com/371943/206781853-f6e13f95-f438-4071-a13c-ad1ae7845843.png">